process-migration-service/frontend/src/component/tabMigrationPlan/MigrationPlansBase.js

```diff
@@ -56,6 +63,11 @@ export default class MigrationPlansBase extends React.Component {
       .then(function() {
         self.retrieveAllPlans();
         self.hideDeleteDialog();
+      })
+      .catch(() => {
```

Comment:
There is no invocation of kie-server in this function, right? The catch block provides wrong explanation, might be redundant.